api/src/paths/user/add.ts

```diff
@@ -111,6 +121,14 @@ export function addSystemRoleUser(): RequestHandler {
       throw new HTTP400('Missing required body param: identitySource');
     }
 
+    if (!displayName) {
+      throw new HTTP400('Missing required body param: identitySource');
+    }
+
+    if (!email) {
```

Comment:
The explicit checks are just older, and likely not needed. There may be some rare situations where a check is needed, because its doing something openapi doesnt support, but for most of them these manual checks can be removed. I mainly just matched the old pattern when I added these 2 new fields, because this particular change is temporary (the user stuff is all going to change real soon, and I just added these to make testing easier).